go-controller/pkg/ovn/kubevirt_test.go

```diff
@@ -1007,6 +1024,71 @@ var _ = Describe("OVN Kubevirt Operations", func() {
 					},
 				},
 			}),
+			Entry("for pre-copy live migration at local zone from remote zone", testData{
```

Comment:
consider please adding to PR desc how it is tested
why 3 nodes ?
and the flow, according the ticket, it should migrate back and forth to the same node etc
if this what happens here (didnt dive yet)

Reply:
Done